go/mysql/conn.go

```diff
@@ -1261,35 +1331,96 @@ func parseEOFPacket(data []byte) (warnings uint16, more bool, err error) {
 	return warnings, (statusFlags & ServerMoreResultsExists) != 0, nil
 }
 
-func parseOKPacket(data []byte) (uint64, uint64, uint16, uint16, error) {
-	// We already read the type.
-	pos := 1
+// PacketOK contains the ok packet details
+type PacketOK struct {
```

Comment:
looks like we are not using this struct outside this package. should it be made internal?